submissions/Harnytskyi/task_js_dom/js/script.js

```diff
@@ -0,0 +1,101 @@
+        img: "img/grumpy-cat.jpg"
+    },
+    {
+        title: "Crying Cat",
+        about: "Crying Cat, also known as Schmuserkadser, refers to a series of photoshopped images of cats with teary, glassy eyes to appear as though they are sad. The earliest known usage of the Crying Cat was published by an anonymous Meme Generator user on June 11th, 2014 (shown below, left). The image is a photoshopped version of Serious Cat, due to the fact that the background of the image and ears and color of the cat are identical (shown below, right). Within four years, the image has generated more than 925 images.",
+        img: "img/crying-cat.jpg"
+    },
+    {
+        title: "Whoosh",
+        about: "Whoosht, known in Russia as вжух, is an image of a cat crudely dressed as a wizard that casts unfortunate spells. It grew popular on Russian websites such as Numl in December of 2016. The image of the cat is taken from a post on justcuteanimals.com (shown below, left). It later gained some traction as \"Wizard Cat\" on me_irl following a post from April 2016 that referred to it as the High Wizard Cat (shown below, right).",
+        img: "img/whoosh-cat.jpg"
+    }
+]
+
+const nav_ul = document.querySelector(".nav_ul");
+const aside = document.querySelector("aside");
+const main = document.querySelector("main");
+let openbtn = document.querySelector(".nav-open-button");
+let navButtonListArray;
+let selectedNav;
+
+function createNavButtons() {
+    let navLi;
+    let navButtonList = document.createDocumentFragment();
+    let navButtonItem;
+
+    nav_ul.innerHTML = "";
+    data.forEach(function (element) {
+        navLi = document.createElement("li");
+        navButtonItem = document.createElement("button");
+        navButtonItem.classList.add("nav-button");
+        navButtonItem.innerHTML = element.title;
+        navLi.append(navButtonItem);
+        navButtonList.append(navLi);
+    });
+    nav_ul.append(navButtonList);
+    navButtonListArray = Array.from(document.querySelectorAll(".nav-button"));
+}
+
+function displayMain({ target }) {
+    if (target.classList == 'nav-button') {
+        highlight(target);
+    }
+    const selectedButtonIndex = navButtonListArray.indexOf(target);
+    let element;
+    main.innerHTML = "";
+
+    element = document.createElement("div");
+    element.classList.add("mainItem-title");
+    element.innerHTML = data[selectedButtonIndex].title;
+    main.append(element);
+    element = document.createElement("img");
+    element.classList.add("mainItem-image");
+    element.src = data[selectedButtonIndex].img;
+    main.append(element);
+    element = document.createElement("div");
+    element.classList.add("mainItem-about");
+    element.innerHTML = "About:<br>" + data[selectedButtonIndex].about;
+    main.append(element);
+
+
+}
+function hideMenu(target) {
+    aside.classList.toggle('hidden');
+}
+
+function highlight(target) {
```

Comment:
Please, use more informative function names. This function name doesn't give any information what the element will be highlighted.